### PR TITLE
Added descriptions for libraries settings.

### PR DESCRIPTION
**Pull request by LiborBatek**
Added descriptions for addon's setting for better clarity and fixed small issue in the info text for OIIO category.

Resolves [https://github.com/ynput/ayon-third-party/issues/11](url)

Testing notes:

1) Build addon package and upload to the server
2) Check addon's settings categories and added descriptions if correct.

### CODE REVIEW

**Review by BigRoy — COMMENTED** [review]
I think these are too verbose and not needed descriptions.

To me just "Configure FFMPEG" and "Configure OIIO" could suffice - but the label of the section already clarifies what it's about. I'm not too bothered with adding it - but it does feel more cluttered to me.

**Comment by LiborBatek** [comment]
I have just started to add those descriptions going addon by addon (from top to bottom) as I like the tips really. 

I get your point but the more info the better imho...speaking of cluttering the UI its more a matter of how its shown then the actual wording I have also seen much longer and clipped descriptions accross AYON and settings....

I have to admit, I would like to have those always as ```hoover over tips``` much more than these header ones tho. But thats something out of my capabalities...

@m-u-r-p-h-y  any opinion on this?

**Review by iLLiCiTiT — APPROVED** [review]

File: server/settings.py [hunk 1]
```diff
@@ -102,8 +102,15 @@ class ThirdPartySettings(BaseSettingsModel):
     ffmpeg: FFmpegSettings = Field(
         default_factory=FFmpegSettings,
         title="FFmpeg",
+        description=(
+            "Configure library tools for transcoding and conversion of media."
+        ),
```
Comment (iLLiCiTiT) [inline server/settings.py:107]: This seems to be unncessary...
```suggestion
```

File: server/settings.py [hunk 2]
```diff
@@ -102,8 +102,15 @@ class ThirdPartySettings(BaseSettingsModel):
...
     )
     oiio: OIIOSettings = Field(
         default_factory=OIIOSettings,
         title="OpenImageIO",
+        description=(
+            "Configure library for image processing and format"
+            " conversion using OIIO."
+        ),
```
Comment (iLLiCiTiT) [inline server/settings.py:115]: This seems to be unncessary...
```suggestion
```

**Pull request closed without merging**